__tests__/app-tests/app.test.js

```diff
@@ -138,4 +150,50 @@ describe('GET /api/articles', () => {
                 descending: true})
         })
     })
+})
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: Responds with an array of comments for the given article_id of which each comment should have the tested properties', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .then(({body}) => {
+            const {comments} = body
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
```

Comment:
you will need a length check for your comments.  if array returned is empty .forEach() will not run an you could end up with a false passing test!